Source/Scene/OrthographicFrustum.js

```diff
@@ -267,6 +269,7 @@ define([
     };
 
     /**
+     * @Deprecated
```

Comment:
For consistency, use `@deprecated`.

Also, did you build the doc to make sure this works?  Some tags like `@private` need to be at the bottom.